packages/keystone/src/___internal-do-not-use-will-break-in-patch/require-source.ts

```diff
@@ -0,0 +1 @@
+export { requireSource } from '../lib/config/requireSource';
```

Comment:
I've intentionally not added a changeset for this, it'll happen to be released at some point but it's not public so I don't want to talk about and imply that it should be used. It's entirely because this is the easiest way to get that code into our script.